containers/flatpak/test/test-browser

```diff
@@ -11,8 +11,23 @@ TESTLIB = os.path.join(TEST_DIR, "test-lib.js")
 FLATPAK_ID = "org.cockpit_project.CockpitClient"
 JS_TIMEOUT = 15
 
+STORAGE = os.path.join(os.path.expanduser("~"), ".var/app/org.cockpit_project.CockpitClient/data/webkitgtk/localstorage/")
```

Comment:
Simplify to

    os.path.expanduser("~/.var/app/org.cockpit_project.CockpitClient/data/webkitgtk/localstorage/")

Reply:
done in a fixup